mappings/net/minecraft/util/registry/RegistryKey.mapping

```diff
@@ -1,47 +1,102 @@
-		COMMENT the value-holding registry in the root registry and an identifier of the
-		COMMENT value.
+		COMMENT Sugarcoats an identifier to a specific usage, given the registry key of
+		COMMENT the registry of that usage.
 		COMMENT
-		COMMENT <p>You can call it like {@code RegistryKey.of(Registry.ITEM_KEY, new Identifier("iron_ingot"))}
-		COMMENT to create a registry key for iron ingot.</p>
+		COMMENT <p>For example, you can sugarcoat an identifier for items with
+		COMMENT {@link Registry#ITEM_KEY the registry key of the item registry}.
+		COMMENT You can call it like {@code RegistryKey.of(Registry.ITEM_KEY, new Identifier("iron_ingot"))}
+		COMMENT to sugarcoat the identifier for iron ingot.</p>
 		COMMENT
-		COMMENT @param <T> the type of the value
+		COMMENT @param <T> the usage type
+		COMMENT @return the sugarcoated identifier
 		ARG 0 registry
-			COMMENT the registry key of the registry in the root registry
+			COMMENT the registry key of the registry of that usage, usually
+			COMMENT available in {@link Registry}
 		ARG 1 value
-			COMMENT the identifier of the value
+			COMMENT the identifier to sugarcoat
 	METHOD method_29180 ofRegistry (Lnet/minecraft/class_2960;)Lnet/minecraft/class_5321;
-		COMMENT Creates a registry key for a registry in the root registry (registry of
-		COMMENT all registries) with an identifier for the registry.
+		COMMENT Sugarcoats an identifier for accessing a specific type of registry in
+		COMMENT the root registry.
+		COMMENT
+		COMMENT <p>You can call it like {@code RegistryKey.ofRegistry(new Identifier("block"))}
+		COMMENT to obtain a registry key of {@code RegistryKey<Registry<Block>>} type
+		COMMENT for the block registry.</p>
 		COMMENT
-		COMMENT <p>You can call it like {@code RegistryKey.of(new Identifier("block"))}
-		COMMENT to create a registry key for the block registry.</p>
+		COMMENT <p><strong>Notice</strong>: This method can refit the generic {@code <T>}
+		COMMENT as any type, so it's on the user's discretion to assign a correct
+		COMMENT type!</p>
 		COMMENT
-		COMMENT @param <T> the element type of the registry
+		COMMENT @param <T> the element type of the registry, where {@code Registry<T>}
+		COMMENT            would be the actual usage of the returned key
+		COMMENT @return the sugarcoated identifier
 		ARG 0 registry
 			COMMENT the identifier of the registry
 	METHOD method_29181 of (Lnet/minecraft/class_2960;Lnet/minecraft/class_2960;)Lnet/minecraft/class_5321;
+		COMMENT Sugarcoats an identifier with a specific usage.
+		COMMENT
+		COMMENT <p>This method handles the identity-based handling of registry keys.</p>
+		COMMENT
+		COMMENT <p>Though this method is private, calling
+		COMMENT {@code RegistryKey.of(RegistryKey.ofRegistry(registry), value)} has the
+		COMMENT same effect.</p>
+		COMMENT
+		COMMENT @param <T> the usage type
 		ARG 0 registry
+			COMMENT the usage
 		ARG 1 value
+			COMMENT the sugarcoated identifier
```

Comment:
"sugarcoated" doesn't sound like the correct word here, maybe you want "wrapped"?